subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/IndexPageGenerator.java

```diff
@@ -65,6 +66,7 @@ public IndexPageGenerator(ResultsStore resultsStore, File resultJson) {
         Comparator<ScenarioBuildResultData> comparator = comparing(ScenarioBuildResultData::isBuildFailed).reversed()
             .thenComparing(ScenarioBuildResultData::isSuccessful)
             .thenComparing(comparing(ScenarioBuildResultData::isAboutToRegress).reversed())
+            .thenComparing(comparing(ScenarioBuildResultData::isCrossBuild).reversed())
```

Comment:
I still don't like the sorting. I think that the cross build results should go into the middle of the report, and not right below the nearly failing scenarios. This is not a deal-breaker for me, though.